sagafalabella/sagafalabella/spiders/falabella.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+import scrapy
+from urllib.parse import quote_plus
+import json
+from scrapy import Request
+
+class FalabellaSpider(scrapy.Spider):
+    name = 'falabella'
+    allowed_domains = ['falabella.com.pe/falabella-pe']
+    start_urls = ['http://falabella.com.pe/falabella-pe']
+
+    def parse(self, response):
+        # we are not interested in some anchors give us url to all categories
+        categories = response.xpath(
+            '//ul[@class="fb-masthead__grandchild-links"]/li/a[not(contains(text(),"Ver todo"))]/@href').extract()
+        for categorie in categories:
+            categorie_url = response.urljoin(categorie)
+            yield Request(categorie_url, callback=self.parse_categorie, dont_filter=True)
+
+    def parse_categorie(self, response):
+        
+        if len(response.xpath('//head/link[@rel="next"]/@href').extract()):
+            nav_state = response.url.replace(
+                'https://www.falabella.com.pe/falabella-pe', '')
+            url_tab = '{"currentPage":2,"navState":"%s"}' % nav_state
+            url_encode = quote_plus(url_tab)
```

Comment:
Nice one! :+1: , maybe you could change `url_tab` to a dictionary as it's more readable.